node/src/GlideClient.ts

```diff
@@ -330,4 +331,33 @@ export class GlideClient extends BaseClient {
     public lolwut(options?: LolwutOptions): Promise<string> {
         return this.createWritePromise(createLolwut(options));
     }
+
+    /**
+     * Loads a library to Valkey.
+     *
+     * See https://valkey.io/commands/function-load/ for details.
+     *
+     * since Valkey version 7.0.0.
+     *
+     * @param libraryCode - The source code that implements the library.
+     * @param replace - Whether the given library should overwrite a library with the same name if it
+     *     already exists.
+     * @returns The library name that was loaded.
+     *
+     * @example
+     * ```typescript
+     * // Example usage of `FUNCTION LOAD` command
```

Comment:
nit: some examples have this line and some do not, but I think its unnecessary/redundant, should we skip it in the future? Unless its necessary to clarify what the example is doing. For most cases it should be obvious enough